app/util/ContentUpgrade.scala

```diff
@@ -48,20 +55,41 @@ object ContentUpgrade {
       val jsonString = JsonMethods.compact(JsonMethods.render(json))
       val maybeParsedJson: Option[Json] = parser.parse(jsonString).right.toOption
       val maybeCapiContent: Option[Content] = maybeParsedJson.flatMap(json => json.as[Content].right.toOption)
-
       (json, maybeCapiContent) match {
         case (jsObject: JObject, Some(content)) =>
           val cardStyle = CardStyle(content, TrailMetaData.empty)
-          val metaDataMap: Map[String, Boolean] = ResolvedMetaData.toMap(ResolvedMetaData.fromContent(content, cardStyle))
+          val initialMetaDataMap: Map[String, Boolean] = ResolvedMetaData.toMap(ResolvedMetaData.fromContent(content, cardStyle))
+
+          // ResolvedMetaData here has a ton of booleans, but some may have been set naively
+          // If the cutout bool is set, we want to find the cutout values and add them in to a sub-object
+          val (metaDataMap, maybeFirstContributorWithCutout) = initialMetaDataMap.get("imageCutoutReplace") match {
+            case Some(true) => {
+              getBylineLargeImageUrlOption(content)  match {
+                case bylineLargeImageUrlOption@Some(_) => (initialMetaDataMap, bylineLargeImageUrlOption)
+                // If we can't find a cutout value, turn off the cutout bool
+                case None => (initialMetaDataMap + ("imageCutoutReplace" -> false), None)
+              }
+            }
+            case _ => (initialMetaDataMap, None)
+          }
```

Comment:
Would be great to extract this block and then factor out https://github.com/guardian/facia-tool/blob/master/app/services/Capi.scala#L180-L187

Reply:
I don't have time today.  Tech debt card?

Reply:
Yes.